packages/make-styles/src/types.test.ts

```diff
@@ -0,0 +1,137 @@
+import { MakeStylesStyle } from './types';
+
+function assertType(style: MakeStylesStyle): MakeStylesStyle {
+  return style;
+}
+
+describe('types tests', () => {
```

Comment:
I'm not a fan of these tests, there's lots of repeated things so it's not easy to tell what is/isn't tested.

I guess you have a larger tests for:
* styles object
* nested objects (pseudos, media, etc...)

Then you also have different styles that you test on:
* flex shrink
* color
* background
* padding

The tests also don't differ too much and simply calls `assertType`. I would rather this test module was refactored so that new test cases are easy to declare

Reply:
Any suggestions how to improve this?

Reply:
I tried to parametrize this, and ended up with a solution with union types that doesn't look too much better :/